package.json

```diff
@@ -43,7 +44,9 @@
     "fakefile": "^0.0.9",
     "jest": "^26.6.3",
     "nock": "^13.0.5",
+    "npm-run-all": "^4.1.5",
```

Comment:
what's this for?

Reply:
Ah, it's to make linting run in parallel to save time, but i didn't wire it up yet. Did that just now 👌